### PR TITLE
Add popper and input for search

### PR DESCRIPTION
**Pull request by ValeryiaMIRON**
- Add popper and input for search

https://trello.com/c/gX3jAskd/118-fe-create-search-for-list-of-candidates

Снимок экрана 2021-11-08 в 11 29 08 PM

### CODE REVIEW

File: src/pages/candidates/Candidates.jsx [hunk 1]
```diff
@@ -77,6 +86,23 @@ export const Candidates = () => {
...
+            <IconButton>
+              <MailOutline fontSize="large" />
+            </IconButton>
+            <IconButton>
+              <Print fontSize="large" />
+            </IconButton>
+          </Box>
+          <Popper open={open} anchorEl={anchorEl} placement="left">
+            <Input
+              placeholder="Search..."
```
Comment (nsukhotskaya) [inline src/pages/candidates/Candidates.jsx:103]: Move text to json file

File: src/pages/candidates/Candidates.jsx [hunk 2]
```diff
@@ -21,6 +23,9 @@ import 'ag-grid-community/dist/styles/ag-theme-alpine.css';
 
 export const Candidates = () => {
   const [gridApi, setGridApi] = useState();
+  const [anchorEl, setAnchorEl] = useState();
+  const open = Boolean(anchorEl);
```
Comment (nsukhotskaya) [inline src/pages/candidates/Candidates.jsx:27]:
```suggestion
  const open = !!anchorEl;
```

File: src/pages/candidates/Candidates.jsx [hunk 3]
```diff
@@ -21,6 +23,9 @@ import 'ag-grid-community/dist/styles/ag-theme-alpine.css';
 
 export const Candidates = () => {
```
Comment (svtslv17) [inline src/pages/candidates/Candidates.jsx:24]:
```suggestion
const Candidates = () => {
```

File: src/pages/candidates/Candidates.jsx (hunk 1 above)
Comment (ValeryiaMIRON) [reply src/pages/candidates/Candidates.jsx]: done

File: src/pages/candidates/Candidates.jsx (hunk 2 above)
Comment (ValeryiaMIRON) [reply src/pages/candidates/Candidates.jsx]: done

File: src/pages/candidates/Candidates.jsx (hunk 3 above)
Comment (ValeryiaMIRON) [reply src/pages/candidates/Candidates.jsx]: done

**Review by nsukhotskaya — APPROVED** [review]

**Review by beliavskaja — APPROVED** [review]

**Pull request merged**